go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/jetstack/cert-manager-webhook-example
 go 1.13
```

Comment:
Do we want to bump this?

Reply:
As this is an example I'd be tempted to leave it alone in case people haven't updated yet for whatever reason.